docs/sad/src/01_introduction_and_goals.adoc

```diff
@@ -1,56 +1,50 @@
-ifndef::imagesdir[:imagesdir: ../images]
-
 [[section-introduction-and-goals]]
 == Introduction and Goals
 
-
-
 === Requirements Overview
 
-What is SIARD-Suite?
+What is SIARD Suite?
 
 * SIARD Suite is a set of tools for the long-term archiving of data stored in relational databases as SIARD archives.
-* SIARD Suite is the reference implementation of the [SIARD Specification](https://siard.dilcis.eu/SIARD%202.2/SIARD%202.2.pdf)
+* SIARD Suite is the reference implementation of the https://siard.dilcis.eu/SIARD%202.2/SIARD%202.2.pdf[SIARD Specification.]
 
 Main Features:
 
 * Support for multiple relational database management systems.
-* Graphical user interface (GUI) and command line interface (CLI) for archiving and restoring data
-* Clients for Windows, Mac and Linux
+* Graphical user interface (GUI) and command-line interface (CLI) for archiving and restoring data.
+* Clients for Windows, Mac, and Linux.
 
 === Quality Goals
-
+// TODO: These are not quality goals. Delete table?
```

Comment:
Quality goals could be a part of the Software Architecture Documentation. The empty table is a reminder that we should set these goals. We could delete the chapter now and add it later if we now the quality goals - or we could add some goals now.